src/Kernel32.Shared/Kernel32.cs

```diff
@@ -208,11 +208,24 @@ public enum FormatMessageFlags
         /// <param name="hObject">A valid handle to an open object.</param>
         /// <returns>
         /// If the function succeeds, the return value is nonzero.
-        /// If the function fails, the return value is zero.To get extended error information, call GetLastError.
+        /// <para>If the function fails, the return value is zero.To get extended error information, call GetLastError.</para>
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool CloseHandle(IntPtr hObject);
 
+        /// <summary>
+        /// Retrieves the thread identifier of the calling thread.
+        /// </summary>
+        /// <returns>The thread identifier of the calling thread.</returns>
+        [DllImport(nameof(Kernel32))]
+        public static extern uint GetCurrentThreadId();
```

Comment:
I agree CLS compliance isn't a goal here. But since BCL uses int everywhere, even when signed integers clearly do not apply (Array.Length, Stream.Position, etc) it makes handling uint elsewhere a real pain. But I'll defer to your judgment -- at least in this case. I'm still divided on the subject myself and have been doing a mix in my contributions. I'll usually stick with the original signature -- unless in the very first test I write I have to use casting anyway and the value will never exceed int.MaxValue such that it would ever possibly be an issue.